content/about/introduction/introduction.html

```diff
@@ -0,0 +1,112 @@
+  </head>
+  <body>
+    <main>
+      <h1>Introduction</h1>
+      <p>
+        This introduction to the ARIA Authoring Practices Guide (APG) explains the purpose of the APG, who it intends to serve, and how it is organized.
+      </p>
+
+      <section id="purpose">
+        <h2>Purpose of the APG</h2>
+        <p>
+          The APG explains how to develop web experiences for people who rely on assistive technologies and keyboard interfaces.
+          It includes a library of accessible user interface patterns, example implementations of the patterns, and guidance that explain how to:
+        </p>
+        <ul>
+          <li>
+            Make common user interface patterns accessible by using features of the
+            <a class="specref" href="#abstract">Accessible Rich Internet Application (ARIA) specification</a>
+            in combination with HTML, CSS, JavaScript, and other web technologies.
+          </li>
+          <li>
+            Use ARIA, HTML, and CSS to implement fundamental accessibility practices, such as providing accessible names, enabling keyboard interaction, conveying page structure to assistive technologies, and supporting high contrast operating system settings.
+          </li>
+        </ul>
+        <p>
+          The primary aim of the APG is to increase the availability of robust accessible experiences by building broad understanding  and encouraging adoption of web accessibility practices that:
+        </p>
+        <ul>
+          <li>Are consistent with the intent of ARIA, HTML, CSS, and related  specifications.</li>
+          <li>Support assistive technology interoperability.</li>
+          <li>Promote practical techniques for developing modern GUIs that both support usability for people with disabilities and conform with WCAG requirements.</li>
+        </ul>
+      </section>
+
+      <section id="apg-is-informative">
+        <h2>APG is Not a Normative Standard</h2>
+        <p>
+          The accessibility guidance in the APG is different from accessibility requirements specified by <a href="https://www.w3.org/TR/WCAG21/">WCAG</a> and <a href="https://www.w3.org/TR/wai-aria/">ARIA</a>.
+          WCAG and ARIA are "W3C normative technical standards" while the APG is an "informative" resource.
+          W3C normative technical standards, which are formally known as "W3C Recommendations", have conformance models that define how to comply with the requirements documented in the standard.
+          Those requirements are developed using a rigorous process defined by
+          The <a href="https://www.w3.org/Consortium/Process#rec-track">W3C Recommendation Track</a>.
+        </p>
+        <p>
+          The APG does not specify normative requirements and thus does not have a conformance model.
+          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href=""https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
```

Comment:
Two double quotes broke the parsing
```suggestion
          The <a href="https://www.w3.org/groups/tf/aria-practices">ARIA Authoring Practices Task Force</a>, which is part of the <a href="https://www.w3.org/groups/wg/aria">ARIA Working Group</a>,
```